packages/editor/src/client/panel/imageEditPanel.tsx

```diff
@@ -238,6 +296,7 @@ export function ImagedEditPanel({id, file, onClose, onSave}: ImageEditPanelProps
                 <DescriptionListItem label={t('images.panels.fileSize')}>
                   {prettyBytes(fileSize)}
                 </DescriptionListItem>
+                {resizeImageHint(file!)}
```

Comment:
I'm not sure if the the `resizeImageHint` is at the right place. If i were uploading images i want to have the information that my image will be compressed above a certain size before i upload it. I suggest to add the hint below the upload field in `packages/editor/src/client/panel/imageUploadPanel.tsx`. Also include the size after which images will be compressed in the hint. Our i18n translation service allows dynamic parameters.

Reply:
I optimized the hint right beneath the image preview before uploading it to the server. Added the expected file size in the translation.